### PR TITLE
Add type coverage check to travis

### PR DESCRIPTION
**Pull request by mandarvaze**
Fixes issue #11

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-pyramid-heroku = {editable = true, path = "."}
+pyramid-heroku = {editable = true,path = "."}
+mypy = "*"
```
Comment (zupo) [inline Pipfile]: Why is mypy listed here? It should be enough to have it only under `[dev-packages]`.

File: pyramid_heroku/migrate.py [hunk 2]
```diff
@@ -15,9 +16,10 @@ class Heroku(object):
     api_endpoint = "https://api.heroku.com"
 
     def __init__(self, app_name, ini_file, app_section):
-        # type: (str, str)-> ()
+        # type: (str, str, str)-> None
```
Comment (zupo) [inline pyramid_heroku/migrate.py]: Please use Python-3 style type hinting that is inline, not in comments.

Comment (mandarvaze) [reply pyramid_heroku/migrate.py]: I just went with the existing style.
Should I make similar changes to rest of the file as well ?

File: Pipfile (hunk 1 above)
Comment (mandarvaze) [reply Pipfile]: Yup. I didn't realize I added it at two places. Removing.

File: pyramid_heroku/migrate.py (hunk 2 above)
Comment (zupo) [reply pyramid_heroku/migrate.py]: Yes.

**Comment by mandarvaze** [comment]
@zupo I've updated the PR and addressed both the comments.

**Pull request merged**